src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php

```diff
@@ -29,6 +29,7 @@ class OutputFormatterStyle implements OutputFormatterStyleInterface
         'magenta' => array('set' => 35, 'unset' => 39),
         'cyan' => array('set' => 36, 'unset' => 39),
         'white' => array('set' => 37, 'unset' => 39),
+        'default' => array('set' => 39, 'unset' => 39),
```

Comment:
According to the standard, it's indeed "default": http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-048.pdf (8.3.117)